### ISSUE TITLE
AoDropdownOption: Not accessible via keyboard

### ISSUE THREAD

**Issue by busheikin** [AmpleOrganics/Blaze.vue#351]
# Issue summary

The dropdown options cannot be accessed by tab navigation on the keyboard, which would be an accessibility violation,

## Expected behavior

Pressing tab allows me to focus on interactive elements

## Actual behavior

Pressing tab doesn't focus on dropdown items

## Specifications

- Blaze version number: 3.4.0
